lib/game_logic.rb

```diff
@@ -18,7 +18,7 @@ def check_outcome(player)
         break
       elsif player.counter >= 5 && intersection != sub_array
         @game_over = true
-        @winner = 'DRAW'
+        @winner = 'VER'
```

Comment:
- [ ] [Optional] please explain why you chose `ver` instead of `draw` in the read me as it is confusing. Or revert back to `draw`

Reply:
ok, thank you.